app/Http/Controllers/ReplyController.php

```diff
@@ -45,6 +46,17 @@ public function delete(Reply $reply)
         return $this->redirectToReplyAble($reply->replyAble());
     }
 
+    public function markAsSpam(Reply $reply)
```

Comment:
Why does this not use the `$request` object like other methods? Is there a reason to not stick with conversion here? 🤔

Reply:
Yeah go for it 👍

Reply:
committed https://github.com/laravelio/laravel.io/pull/886/commits/b17864d1179a90e21176db8edf367d66df491f05